src/core/structure/sheet.ts

```diff
@@ -62,25 +79,37 @@ export default class Sheet {
 
     if (cell) {
       cell.formula = value;
-      this.resolveCell(cell);
+      this.resolveCell(cell, colKey, rowKey);
     }
```

Comment:
Some refactoring while reviewing the code

```suggestion
    if (cell) {
      if (value == "") {
        // Cell exists but is being cleared.
        this.deleteCell(colKey, rowKey);
      }
      cell.formula = value;
      this.resolveCell(cell, colKey, rowKey);
    } else {
      cell = this.createCell(colKey, rowKey, value);
    }
```

Reply:
This removes the `resolveCell` call for newly created cells which I think we should be doing. The default formatter could modify the value of the new cell even if its contents are empty.